src/GraphQL/Conversion/ValueConverter.cs

```diff
@@ -37,6 +37,10 @@ static ValueConverter()
             Register<string, float>(value => float.Parse(value, NumberStyles.Float, NumberFormatInfo.InvariantInfo));
             Register<string, double>(value => double.Parse(value, NumberStyles.Float, NumberFormatInfo.InvariantInfo));
             Register<string, decimal>(value => decimal.Parse(value, NumberStyles.Float, NumberFormatInfo.InvariantInfo));
+#if NET6_0_OR_GREATER
+            Register<string, DateOnly>(value => DateOnly.Parse(value, DateTimeFormatInfo.InvariantInfo));      
```

Comment:
No DateTimeStyles ?

Reply:
dateonly has no timezone, hence no universal time, so `DateTimeStyles.AdjustToUniversal | DateTimeStyles.AssumeUniversal` doesnt make sense. should i have included other DateTimeStyles?

Reply:
I don't know. I just see that `Parse` method accepts `DateTimeStyles` argument.

Reply:
agreed. also i cant find any docs about how DateTimeStyles works in the context of DateOnly

Reply:
So maybe use `DateTimeStyles.None` explicitly to highlight this decision and go on?